hudi-common/src/test/java/org/apache/hudi/common/model/HoodieTestUtils.java

```diff
@@ -96,21 +96,20 @@ public static HoodieTableMetaClient init(String basePath)
   }
 
   public static HoodieTableMetaClient init(String basePath, HoodieTableType tableType) throws IOException {
-    return initTableType(getDefaultHadoopConf(), basePath, tableType);
+    return init(getDefaultHadoopConf(), basePath, tableType);
   }
 
   public static HoodieTableMetaClient init(Configuration hadoopConf, String basePath)
       throws IOException {
-    return initTableType(hadoopConf, basePath, HoodieTableType.COPY_ON_WRITE);
+    return init(hadoopConf, basePath, HoodieTableType.COPY_ON_WRITE);
   }
 
-  public static HoodieTableMetaClient initTableType(Configuration hadoopConf, String basePath,
-      HoodieTableType tableType) throws IOException {
+  public static HoodieTableMetaClient init(Configuration hadoopConf, String basePath, HoodieTableType tableType) throws IOException {
```

Comment:
Line is longer than 120 characters (found 133). [LineLength]

Reply:
@leesf Thanks for reminding me. I have fixed the checkstyle issue.